paint/PAINT_Screen.cpp

```diff
@@ -39,75 +39,78 @@ Screen::Screen(SDL_Renderer* renderer, const gfx::Rectangle& rect, const char* n
 	this->addChild(toolWindow);
```

Comment:
This function is necessarily quite long, and motivates a data-driven screen layout. Eg store the structure of the screen in an XML file that is loaded at runtime. This file can be created by an editor tool that is itself written using our GUI library :-) 
**MIGHT**